src/context/json.ts

```diff
@@ -7,12 +12,19 @@ import { ResponseTransformer } from '../response'
  * res(json('Some string'))
  * res(json([1, '2', false, { ok: true }]))
  */
-export const json = <BodyType>(
-  body: BodyType,
-): ResponseTransformer<BodyType> => {
+export const json = <BodyTypeJSON>(
+  body: BodyTypeJSON,
+  { merge = false }: JSONContextOptions = {},
+): ResponseTransformer<BodyTypeJSON> => {
   return (res) => {
     res.headers.set('Content-Type', 'application/json')
-    res.body = JSON.stringify(body) as any
+
+    if (merge) {
```

Comment:
This `if` is unnecessary: you can assign a ternary expression to the `res.body`

```js
res.body = merge ? mergeRight(...) : body
```

That produces the same output as the if/else statement.

Reply:
True.